.github/workflows/build.yml

```diff
@@ -8,6 +8,9 @@ on:
       - 'trunk'
   pull_request:
 
+permissions:
+  contents: none
```

Comment:
(Forgot to save this comment.)  Quoting from the doc:

> If you specify the access for any of these scopes, all of those that are not specified are set to none

So, the two lines above actually set all permissions to `none`.  That's on the safe side, great!  But that's not explicit at all...  Quoting the doc again:

> You can use the following syntax to disable permissions for all of the available scopes: `permissions: {}`

Could this be a clearer way to express what is meant here?

Reply:
It does indeed - I've used that and added a comment explaining why it's there